python/requirements.txt

```diff
@@ -1,5 +1,5 @@
 msgpack-python>=0.5.6
 autowrap>=0.16.0
-pytest
+pytest<5
```

Comment:
Why do we need this change ?

Reply:
see https://github.com/pytest-dev/pytest/issues/5519

pytest dropped support for python 3.4

This fixes it for now, we might want to drop python 3.4 as well but did not wanted to do this as part of this PR